CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @andreeaflorescu @bonzini @jiangliu @tkreuzer
```

Comment:
I don't think I should be the owner of this one. I haven't contributed at all. I believe code owners should be people with a very good understanding of the code.
I would propose you only add @bonzini @jiangliu (maybe @alexandruag if he also wants to maintain the Linux side of it) and @tkreuzer for the Windows side of things.